src/generate.js

```diff
@@ -84,11 +85,20 @@ self.generate = function(options) {
 	if (options.symbols) {
 		pool += symbols;
 	}
+
 	// similar characters
 	if (options.excludeSimilarCharacters) {
 		pool = pool.replace(similarCharacters, '');
 	}
 
+	// excludes characters from the pool
+	if (options.excludes != null) {
```

Comment:
Could we avoid an extra if block if we set `options.excludes` default to an empty string above? (and in docs) Thoughts?